### DISCUSSION TITLE
How can I get the precision, recall, map@0.5, @0.95 of the network prediction？

### DISCUSSION THREAD

**Discussion by NikaisnotRubber** [obss/sahi#863] [Q&A]
In order to detect small targets, I used a lot of 640x640 slice images to train yolov8s（labels are yolov5 format）in my customize dataset, and then used sahi to take the high-resolution images synthesized for detection. The prediction results are good, but I can't find the way to get the precision, recall, map@0.5, @0.95 to show the Improvement. Can I use sahi to detect multiple high-resolution images at once and get its evaluation, precision, recall, map@0.5, map@0.5: 0.95？

**Comment by Grenaf** [comment]
Maybe you need this message，https://github.com/obss/sahi/discussions/718
